server.js

```diff
@@ -39,6 +45,33 @@ function Weather(forecast, time) {
   this.forecast = forecast;
   this.time = time;
 }
+function Event(link, name, event_date, summary) {

```

Comment:
This could be refactored to take in the single parameter since all of these values are coming from the API response. Then you can move all the logic of picking out the values you need into the constructor function. This will make it easier to find that logic later if it needs to be updated, and generally improve how easy it is to read your code.